src/picknik_ur_multi_arm_config/objectives/reset_plannning_scene.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Just noticed that this file has an extra "n" in it.

`reset_plannning_scene.xml`

Reply:
I'll fix it :sweat_smile: